Pod/Classes/MWPhotoBrowser.m

```diff
@@ -474,7 +477,11 @@ - (void)restorePreviousNavBarAppearance:(BOOL)animated {
 
 - (void)viewWillLayoutSubviews {
     [super viewWillLayoutSubviews];
-    [self layoutVisiblePages];
+    if (@available(iOS 11.0, *)) {
+        // do nothing
```

Comment:
add a comment saying that for iOS 11 `layoutVisiblePages` will be called before `viewWillLayoutSubviews`, because of a bug fix

Reply:
there is no need thats called before or after 
it should called only once 
It looks like that in iOS 11 the removeFromSuperview function will call every time viewWillLayoutSubviews.

Reply:
ah ok thanks